test/src/test/java/org/corfudb/integration/CorfuStoreBrowserEditorIT.java

```diff
@@ -200,12 +200,73 @@ public void browserTest() throws
 
         // Invoke tableInfo and verify size
         Assert.assertEquals(browser.printTableInfo(namespace, tableName), one);
-        // Invoke dropTable and verify size
+        // Invoke clearTable and verify size
         Assert.assertEquals(browser.clearTable(namespace, tableName), one);
         // Invoke tableInfo and verify size
         Assert.assertEquals(browser.printTableInfo(namespace, tableName), 0);
     }
 
+    /**
+     * Create a table and add data to it.  Verify that the browser tool is able
+     * to drop that table and all its associated un-shared metadata.
+     * @throws IOException
+     * @throws NoSuchMethodException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     */
+    @Test
```

Comment:
Suggestion: Is it good to add a test that asserts the exception when a table is deleted and is opened before compaction?

Reply:
You mean deleted and opened _after_   compaction right?
opening a table before compaction is a trivial case

Reply:
Yes, after compaction!